core/service.go

```diff
@@ -60,8 +62,19 @@ func (s *Service) start(e chan error) {
 		switch msgType {
 		case p2p.TransactionMsgType:
 			// process tx
+			enc, err := msg.Encode()
+			if err != nil {
+				log.Error("core service", "error", err)
+			}
+			err = s.ProcessTransaction(enc)
```

Comment:
This doesn't really work since there are multiple txs in a message.

Reply:
I will remove this for now, I haven't finished testing it yet